### PR TITLE
bootstrap CI

### PR DESCRIPTION
**Pull request by dericcrago**
Initial migration of `fortios` content out of `community.network` and into `community.fortios`. This includes (what I hope to be) the minimal amount of changes to get the CI working with GitHub actions.

I will follow up with an additional PR for the additional needed files (`README.md`, `galaxy.yml`, etc.), but I thought this would make it easier to review and get the CI going.

### CODE REVIEW

File: .github/workflows/ansible-test.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+# 1. If you don't have unit tests remove that section
+# 2. If your collection depends on other collections ensure they are installed, see "Install collection dependencies"
+# If you need help please ask in #ansible-devel on Freenode IRC
+
+name: CI
+on:
+  # Run CI against all pushes (direct commits, also merged PRs), Pull Requests
+  push:
+  pull_request:
+  # Uncomment the following two lines to run CI once per day (at 06:00 UTC)
```
Comment (dmsimard) [inline .github/workflows/ansible-test.yml:11]: nit, that line can be removed

**Comment by dmsimard** [comment]
@dericcrago  LGTM, unless I can't find it, needs a galaxy.yml at the root

File: .github/workflows/ansible-test.yml (hunk 1 above)
Comment (gundalow) [reply .github/workflows/ansible-test.yml:11]: Given the checklist discussion can you update `collection_template` so the nightly job always runs

**Review by gundalow — COMMENTED** [review]
Looks good, thank you

File: .github/workflows/ansible-test.yml (hunk 1 above)
Comment (dericcrago) [reply .github/workflows/ansible-test.yml]: you already had made that change, my `cookiecutter` was outdated 😞

**Pull request merged**